documentation/source/specific.rst

```diff
@@ -27,34 +32,10 @@ Namespaced Function Examples
 
 .. cpp:namespace:: @ex_specific_namespaced_function
 
-.. doxygenfunction:: testnamespace::NamespacedClassTest::function
-   :path: ../../examples/specific/class/xml
-
-.. doxygenfunction:: testnamespace::ClassTest::function
-   :path: ../../examples/specific/class/xml
-
-.. doxygenfunction:: testnamespace::ClassTest::anotherFunction
-   :path: ../../examples/specific/class/xml
-
-.. doxygenfunction:: ClassTest::function
-   :path: ../../examples/specific/class/xml
-
-.. doxygenfunction:: ClassTest::anotherFunction
-   :path: ../../examples/specific/class/xml
-
-.. doxygenfunction:: f0
-   :path: ../../examples/specific/class/xml
-.. doxygenfunction:: f0< std::string >
-   :path: ../../examples/specific/class/xml
```

Comment:
These are duplicates. They are used in the "Template Type Alias" section on  the same page. I can revert all other namespaced function examples except these.

Reply:
They shouldn't be duplicates, the first finds the primary template and the second a template specialization. I can't immediately find any warnings/errors related to them when I compile, do you have some log output for the problem? (perhaps we should rename the function, ``f0`` appears in a log of hashes)

Reply:
No, I meant that restoring them to this section of the page would create duplicates (see https://github.com/michaeljones/breathe/pull/816/files/21a32d024c0143a7d86a573cfc2a02d182ac759b#r820201537). I understand the difference in the 2 signatures...

Reverting now.

Reply:
Ah, I thought you meant "duplicates of each other" :-).
Thanks for untangling the reuse of files in different examples.